index.js

```diff
@@ -0,0 +1,17 @@
+const fse = require("fs-extra");
+a=fse.readFileSync("input.json","utf8")
+arr=JSON.parse(a);
```

Comment:
1. Add space between the assignment operators. Example: a` = ` b;
2. Also, make sure to add the semicolons `;`.
3. Use [readJsonSync](https://github.com/jprichardson/node-fs-extra/blob/master/docs/readJson-sync.md) instead of `readFileSync` for JSON-type files. If you use `readJsonSync`, you don't need to use `JSON.parse`.